tests/topotests/multicast_mld_join_topo1/test_multicast_mld_local_join.py

```diff
@@ -0,0 +1,1027 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+#
+# Copyright (c) 2023 by VMware, Inc. ("VMware")
+# Used Copyright (c) 2018 by Network Device Education Foundation,
+# Inc. ("NetDEF") in this file.
+#
+# Permission to use, copy, modify, and/or distribute this software
```

Comment:
Drop this text, please see other examples.

Reply:
Refer https://github.com/FRRouting/frr/pull/12780/files

Reply:
I have addressed this comment.